src/apis/klerosapi.py

```diff
@@ -0,0 +1,44 @@
+"""
+KlerosAPI for fetching the Kleros token list.
+"""
+# pylint: disable=logging-fstring-interpolation
+
+import requests
+from src.helper_functions import get_logger
+from src.constants import (
+    header,
+    REQUEST_TIMEOUT,
+)
+
+
+class KlerosAPI:
+    """
+    Class for fetching the Kleros token list.
+    """
+
+    def __init__(self) -> None:
+        self.logger = get_logger()
+
+    def get_token_list(self) -> list[str]:
+        """
+        Returns the Kleros token list.
+        """
+        kleros_url = "http://t2crtokens.eth.link"
```

Comment:
Are you sure this is the correct address? I cannot access it at all.

Reply:
It is, but doesn't seem to be working now. Let me try to add a fallback list there so that the chance of this failing is small.